ui/pages/providers/add.js

```diff
@@ -187,37 +186,39 @@ export default function ProvidersAddDetails() {
                   )
               )}
             </div>
-            <label className='text-2xs font-medium text-gray-700'>Name</label>
-            <input
-              required
-              type='text'
-              value={name}
-              onChange={e => {
-                setName(e.target.value)
-                setErrors({})
-                setError('')
-              }}
-              className={`mt-1 block w-full rounded-md shadow-sm focus:border-blue-500 focus:ring-blue-500 sm:text-sm ${
-                errors.name ? 'border-red-500' : 'border-gray-300'
-              }`}
-            />
-            {errors.name && <ErrorMessage message={errors.name} />}
           </div>
           <div className='w-full'>
             <div className='mb-1 flex'>
               <h3 className='text-base font-medium leading-6 text-gray-900'>
-                Additional Information
+                Information
               </h3>
-              &nbsp;
+              &nbsp; &nbsp;
```

Comment:
i think it looks like there is an extra space on UI. what we can do is to put the `space-x-2` on the <div /> on line 191 to the spacing looks nice